src/transformers/models/mamba2/convert_mamba2_ssm_checkpoint_to_pytorch.py

```diff
@@ -38,7 +38,13 @@ def load_state_dict_from_safetensors(mamba2_checkpoint_path: str, ckpt_name: str
 
 
 def load_state_dict_from_torch(mamba2_checkpoint_path: str, ckpt_name: str) -> Dict[str, torch.Tensor]:
-    return torch.load(path.join(mamba2_checkpoint_path, ckpt_name), map_location="cpu")
+    original_state_dict = torch.load(path.join(mamba2_checkpoint_path, ckpt_name), map_location="cpu")
+    key_map = {
+        'backbone.embedding.weight': 'backbone.embeddings.weight',
```

Comment:
im wondering if this function was ever tested, because how did this rename go undetected?